ci/scripts/integration_substrait.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+# check that optional pyarrow modules are available
+# because pytest would just skip the substrait tests
+echo "Substrait Integration Tests";
+python -c "from substrait_consumer.consumers import AceroConsumer"
+python -c "import pyarrow.orc"
+python -c "import pyarrow.substrait"
```

Comment:
I see the comment but I'm still confused what is happening here.  Is the problem that the consumer testing suite would silently skip all the tests and look like a pass if these commands fail?  I'm pretty sure `pytest` fails if no tests were found.